js/main.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line strict
+'use strict';
+
+const getRandomInt = (num1, num2) => {
+  if (num1 > num2) {
+    return Math.floor(Math.random() * (num1 - num2 + 1) + num2);
+  }
+  return Math.floor(Math.random() * (num2 - num1 + 1) + num1);
+};
+
+getRandomInt(10, 20);
+
+
+const getRandomFloat = (num1, num2, decimals) => {
+  if (num1 > num2) {
+    return +(Math.random() * (num1 - num2) + num2).toFixed(decimals);
+  }
+  return +(Math.random() * (num2 - num1) + num1).toFixed(decimals);
```

Comment:
```suggestion
  if (num1 > num2) {
    return +(Math.random() * (num1 - num2) + num2).toFixed(decimals);
  }
  
  return +(Math.random() * (num2 - num1) + num1).toFixed(decimals);
```

после блока if, если нет else, принято ставить пустую строку. Ну и как в прошлом случае, ты можешь сделать две переменные, в которых ты сделаешь проверки, больше или меньше начальные, поменяешь местами и потом их в результирующее выражение и поставишь